src/main/java/org/candlepin/guice/CandlepinContextListener.java

```diff
@@ -405,11 +424,16 @@ private void insertValidationEventListeners(Injector injector) {
      * @throws RuntimeException if there are missing changesets or a LiqubaseException
      */
     protected void checkDbChangelog() {
-        if (!config.getBoolean(HALT_ON_LIQUIBASE_DESYNC)) {
+        log.info(String.format("Liquibase startup management set to %s",
```

Comment:
format called inside the log call which already has formatting

This could also come after the string below to use the fetched value instead of looking it up twice.